wordpress.org/public_html/wp-content/themes/pub/wporg-support-2024/functions.php

```diff
@@ -16,26 +16,77 @@
  */
 add_filter( 'bbp_show_lead_topic', '__return_true' );
 
+/**
+ * Register a local nav menu for non-English forums, if it doesn't already exist.
+ */
+function register_local_nav_menu() {
```

Comment:
Added this @ryelle but it doesn't seem to work, any ideas? When I debug it seems to be called correctly but is never added to the menu locations array.